LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -9,18 +9,16 @@ function logDebug(callback)
 
 function pause(duration) {
     return new Promise(resolve => {
-        setTimeout(() => {
-            resolve();
-        }, duration);
+        setTimeout(resolve, duration);
     });
 }
 
-function dispatchMouseActions(actions, pointerType)
+async function dispatchMouseActions(actions, pointerType)
 {
     if (!window.eventSender)
-        return Promise.reject(new Error("window.eventSender is undefined."));
+        throw new Error("window.eventSender is undefined.");
 
-    return new Promise(resolve => {
+    await new Promise((resolve, reject) => {
         setTimeout(async () => {
             eventSender.dragMode = false;
 
```

Comment:
Can we flatten this now that we're using async await?

```
 // Wait for tick
await new Promise(resolve => setTimeout(resolve, 0));
eventSender.dragMode = false;
for (let action of actions) {
    ...
    throw new Error(`Unknown action type "${action.type}".`));
...
```

Reply:
yeah, much nicer!